core/src/test/java/com/yandex/yoctodb/SimpleDatabaseTest.java

```diff
@@ -435,6 +450,18 @@ public void stored() throws IOException {
                         .withField("sorted", 12, SORTABLE)
                         .withField("stored", 13, STORED)
                         .withField("first", 14, STORED)
+                        .withField("long_stored", 15L, STORED)
```

Comment:
If you don't mind let's group `int`, `long`, etc. together (here and below). It looks like intentional interleaving, doesn't it?

Reply:
Grouped them by type.